config/sidekiq.yml

```diff
@@ -2,3 +2,7 @@
   - default
   - mailers
   - mail_handlers
+  -
+schedule:
+  UserCleanupJob:
+    cron: '0 8 1 * *'  # At 08:00:00 on each first of the month => https://crontab.guru/#0_8_1_*_*
```

Comment:
I want to suggest at around 10 or 11, since it will be read better (I assume). Also it won't clash with backups then.